metis-media-service/src/test/java/eu/europeana/metis/mediaprocessing/extraction/MediaExtractorImplTest.java

```diff
@@ -70,7 +71,7 @@ static void prepare() {
     textProcessor = mock(TextProcessor.class);
     media3dProcessor = mock(Media3dProcessor.class);
     mediaExtractor = spy(new MediaExtractorImpl(resourceDownloadClient, mimeTypeDetectHttpClient,
-        tika, imageProcessor, audioVideoProcessor, textProcessor, media3dProcessor));
+        new TikaWrapper(), imageProcessor, audioVideoProcessor, textProcessor, media3dProcessor));
```

Comment:
`tika` is never used. I think perhaps here you'd be better off mocking TikaWrapper? Or otherwise just removing the tika value from this class.

Reply:
Forgot to fix that, it is done